### DISCUSSION TITLE
psql in https://app.supabase.com/project/zgyaeycymojyrwunbodq/sql

### DISCUSSION THREAD

**Discussion by sreenik250** [supabase/supabase#8642] [Questions] [answered]
\copy NP_MONEY_CONTROL_CONS_QTR FROM 'C:\AShares\NP_MONEY_CONTROL_CONS_QTR_TMP.csv' DELIMITER ',' CSV HEADER;

The above command runs in my local postgresql.  I would like to execute in my above link.  I am new to supabase.

**Comment by steve-chavez** [comment, marked as answer]
You can open an issue requesting [psql slash commands](https://www.linuxtopia.org/online_books/database_guides/Practical_PostgreSQL_database/c12119_002.htm#:~:text=There%20are%20several%20slash%20commands,most%20of%20those%20same%20settings.) as a feature. Not sure if it's feasible though.

**Reply by sreenik250** [reply]
Any other feature to import CSV file into supabase tables?

**Reply by steve-chavez** [reply]
Yeah, you can do so through the UI 

- https://supabase.com/docs/guides/database#additional-features
- https://medium.com/@isaacyong/how-to-import-csv-into-supabase-20d71e3dfa55

**Reply by sreenik250** [reply]
Connected directly from DBeaver and imported.  Thank you.

**Comment by J0** [comment]
Hey @sreenik250,

Don't think you can currently run slash commands from the UI. Let's continue the discussion on the issue that has been opened: https://github.com/supabase/supabase/issues/8657
